rust/sealevel/programs/install-solana-1.14.20.sh

```diff
@@ -0,0 +1,202 @@
+#!/bin/sh
+
+# Solana CLI releases were migrated from https://release.solana.com to
+# https://release.anza.xyz, but along the way some old versions weren't ported over.
+# This includes 1.14.20, which is the version we require to build the programs.
+
+# This script is adapted from https://release.anza.xyz/v1.18.18/install with the following
+# changes:
+# - The version is hardcoded to 1.14.20
+# - The URL is hardcoded to read from the old solana-labs repo
+# - All agave references are replaced with solana
+
+SOLANA_RELEASE=v1.14.20
+SOLANA_INSTALL_INIT_ARGS=v1.14.20
```

Comment:
maybe a nit but are both needed, just reuse SOLANA_RELEASE?

Reply:
this is what they do in the official install script, I think it's probably better to minimize the diff with what they do even if it means there's redundant code

Reply:
yeah thought as much, thought i'd ask anyway